controllers/config/config.go

```diff
@@ -0,0 +1,15 @@
+package config
+
+var Config config
+
+var (
+	MetricsAddr             string
+	HelmClientRepoCachePath string
+	HelmClientRepoConfPath  string
+	EnableLeaderElection    bool
+)
+
+func FromViper() {
+	Config.HelmClientRepositoryCachePath = HelmClientRepoCachePath
```

Comment:
I guess `Config` can be dropped, if the PFlags are directly assigned to the public vars in this package.

Reply:
If we want to keep being able to read these vars from the environment, we should call `viper.Get()` here (see my comment on `main.go`).